src/window.h

```diff
@@ -71,6 +71,8 @@ class Window : public DMainWindow
 
     void clearBlack();
 
+    QString blankFileDir();
```

Comment:
This could be `const`
```suggestion
    QString blankFileDir() const;
```